test/t8_cmesh/t8_gtest_cmesh_partition.cxx

```diff
@@ -69,14 +61,11 @@ test_cmesh_commited (t8_cmesh_t cmesh)
 
 TEST_P (t8_cmesh_partition_class, test_cmesh_partition_concentrate)
 {
-  /* This if statement is necessary to make the test work by avoiding specific cmeshes which do not work yet for this test.
-   * When the issues are gone, remove the if statement. 
-   */
 
 #ifdef T8_ENABLE_LESS_TESTS
-  const int           level = 6;
+  const int           level = 4;
 #else
-  const int           level = 11;
+  const int           level = 7;
```

Comment:
Does the CI runs out of time for the old configuration of the levels? If not, we should use the higher levels.